src/rotational.jl

```diff
@@ -68,13 +64,14 @@ endpoint_conditioned_sample(rng::AbstractRNG, process::RotDiffusionProcess, s::R
 
 values(r::MultiRotationState) = copy(r.rots)
```

Comment:
I don't think we need this anymore? It was used in the old tracking system, but now the rotations themselves should be returned by the tracker anyway.

Reply:
Agreed. I'll drop MultiRotationState and its functions.